app/src/main/java/com/example/tmdb/screens/home/HomeScreen.kt

```diff
@@ -0,0 +1,163 @@
+package com.example.tmdb
+
+
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.unit.dp
+import com.example.tmdb.modules.FavoritesViewModel
+import com.example.tmdb.modules.HomeViewModel
+import com.example.tmdb.screens.detail.DetailPage
+import com.example.tmdb.screens.home.HomeScreenElements
+import com.example.tmdb.screens.home.MovieCard
+import com.example.tmdb.screens.home.SearchCard
+import com.example.tmdb.screens.home.Tabs
+
+@Composable
+fun HomeScreen(logo : Painter,
```

Comment:
This screen is much more complicated then it needs to be, but let's leave it like this for now. :)